Make things easier to setup (#460)
KMKfw/kmk_firmware: docs/porting_to_kmk.md

]
```

Note: All number are not necessarily used ! The keyboard assume
`number of line * number of rows` keys. Some of the possible keys might not be
used. For example a keyboard with 60 keys might have 8 rows, 8 cols, allowing
64 total combination hence 64 keys. 4 numbers will then not be used for keys
in the `coord_mapping` (might be anyone of them depending of the wiring).

### Find your coord mapping
The following code will help you setup your `coord_mapping` by having every key

> send it's corresponding number. Use it after your pins and module definition
> to define both `keyboard.coord_mapping` and `keyboard.keymap`.

Reviewer: ```suggestion
send its corresponding number. Use it after your pins and module definition
to define both `keyboard.coord_mapping` and `keyboard.keymap`.
```

```python
from kmk.handlers.sequences import simple_key_sequence
from kmk.keys import KC

> # *2 for split keyboard, who will typically managed twice the number of keys
> # of one side. Having this N too much will have no impact (maybe slower boot..)

Reviewer: ```suggestion
# *2 for split keyboards, which will typically manage twice the number of keys
# of one side. Having this N too large will have no impact (maybe slower boot...)
```
